db/system_distributed_keyspace.cc

```diff
@@ -526,11 +529,10 @@ future<std::vector<db_clock::time_point>>
 system_distributed_keyspace::get_cdc_desc_v1_timestamps(context ctx) {
     std::vector<db_clock::time_point> res;
     co_await _qp.query_internal(
-            format("SELECT time FROM {}.{}", NAME, CDC_DESC_V1),
-            quorum_if_many(ctx.num_token_owners),
             // This is a long and expensive scan (mostly due to #8061).
             // Give it a bit more time than usual.
-            get_timeout_config(std::chrono::seconds(60)),
```

Comment:
For consistency - since everyone uses an especially-crafted `query_state` - we could make a function, `get_query_state(duration)` and use `get_query_state(seconds(60))` here, and define `internal_distributed_query_state` as `get_query_state(seconds(10))`.

But that's just a personal preference, so LGTM.